cmds/core/io/smn.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 the u-root Authors. All rights reserved.
```

Comment:
The test is upset you added a period on the end of this line.

Reply:
which is odd as that was copypasted.